src/NServiceBus.Core/Pipeline/Outgoing/OutgoingPipelineFeature.cs

```diff
@@ -21,6 +21,7 @@ protected internal override void Setup(FeatureConfigurationContext context)
             context.Pipeline.RegisterConnector<OutgoingPhysicalToRoutingConnector>("Starts the message dispatch pipeline");
             context.Pipeline.RegisterConnector<RoutingToDispatchConnector>("Decides if the current message should be batched or immediately be dispatched to the transport");
             context.Pipeline.RegisterConnector<BatchToDispatchConnector>("Passes batched messages over to the immediate dispatch part of the pipeline");
+            context.Pipeline.Register("LogOutgoingMessages", typeof(LogOutgoingMessageBehavior), "Ensures logging of the message");
             context.Pipeline.Register("ImmediateDispatchTerminator", typeof(ImmediateDispatchTerminator), "Hands the outgoing messages over to the transport for immediate delivery");
```

Comment:
Would it be better to create the logger here instead and only add the behavior if `log.IsDebugEnabled` ? That would mean less behaviors on the pipeline and shorter callstacks 

cc @danielmarbach